src/documentation/print-linter-wiki.ts

```diff
@@ -29,17 +18,26 @@ import { joinWithLast } from '../util/text/strings';
 import { guard } from '../util/assert';
 import { writeWikiTo } from './doc-util/doc-print';
 import { getFunctionsFromFolder } from './doc-util/doc-functions';
+import { LintingResultCertainty, LintingRuleCertainty } from '../linter/linter-format';
 
-const SpecialTagColors: Record<string, string> = {
+const SpecialBadgeColors: Record<string, string> = {
```

Comment:
I explicitly did not want this to be a badge as a badge should be the thing the linter "certifies" not what it is